src/components/Admin/Module/DalSettings.vue

```diff
@@ -218,6 +242,18 @@ export default {
           return enc
         }, {})
     },
+
+    applySelectedSystemFields (selectedOption) {
```

Comment:
Since this function will now accept the groupType and true/false, you'll probably have to map through the array and if its part of the group set give it omit based on the value

Reply:
the `systemFieldEncoding` is an object and not an array, that't why I used `systemFields` the map over it and return the value